test/unit/retabled/filter_test.cljc

```diff
@@ -0,0 +1,63 @@
+(ns retabled.filter-test
+  (:require [clojure.test :refer [deftest is testing]]
+            [retabled.filter :as filter]))
+
+(deftest resolve-filter
+  (let [resolve-filter-test (fn [expected column entry]
+                              (is (= expected (filter/resolve-filter column entry))))
+        resolve-filter-test-error (fn [column entry]
+                                    (is (thrown-with-msg? Exception #"Unable to resolve filter" (filter/resolve-filter column entry))))]
```

Comment:
Let's make a top-level map that includes error strings and tests like this can just grab the error string and use that, rather than hard-coding strings